shared/src/commonMain/kotlin/co/touchlab/kampkit/Koin.kt

```diff
@@ -23,11 +23,13 @@ fun initKoin(appModule: Module): KoinApplication {
 
     // Dummy initialization logic, making use of appModule declarations for demonstration purposes.
     val koin = koinApplication.koin
-    val doOnStartup = koin.get<() -> Unit>() // doOnStartup is a lambda which is implemented in Swift on iOS side
+    // doOnStartup is a lambda which is implemented in Swift on iOS side
+    val doOnStartup = koin.get<() -> Unit>()
```

Comment:
Was this a ktlint thing? Why is it only coming up now?

Reply:
when I did `shared:build`, `ktlintCheck` failed due to these comment white-spacing issue. Not sure why it never failed before. I failed on my local machine